examples/redis_streams.js

```diff
@@ -0,0 +1,41 @@
+const IORedis = require('ioredis')
+const { promisify } = require('util')
+const redis = new IORedis()
+// this to make them work with await, however you could still
+// use the async version with .then
+const xadd = promisify(redis.xadd).bind(redis)
```

Comment:
ioredis has built-in support for promise so I think we can just use `await redis.xadd(xxx)` right?

Reply:
@luin  yes you are right, fixed